app/src/main/res/layout/fragment_new_detail.xml

```diff
@@ -1,58 +1,83 @@
-            <TextView
-                android:layout_marginStart="@dimen/spacing_medium"
-                android:layout_marginTop="@dimen/static_spacing_medium"
-                android:layout_width="wrap_content"
-                android:layout_height="wrap_content"
-                android:textStyle="bold"
-                android:textSize="@dimen/font_huge"
-                android:textColor="@color/fullWhite"
-                android:text="Post"
-                />
+            android:layout_height="@dimen/new_details_container_height"
+            android:background="@drawable/important"
+            android:orientation="vertical">
+            <LinearLayout
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content">
+                <ImageButton
+                    android:id="@+id/goBack"
+                    android:layout_width="@dimen/back_button_width"
+                    android:layout_height="@dimen/back_button_height"
+                    android:layout_marginStart="@dimen/static_spacing_medium"
+                    android:background="@color/transparent"
+                    android:src="@drawable/ic_baseline_arrow_back_24" />
+                <TextView
+                    android:id="@+id/goBackText"
+                    android:layout_marginStart="@dimen/spacing_medium"
+                    android:layout_marginTop="@dimen/static_spacing_medium"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:textStyle="bold"
+                    android:textSize="@dimen/font_huge"
+                    android:textColor="@color/fullWhite"
+                    android:text="@string/go_back"
+                    />
+            </LinearLayout>
+            <LinearLayout
+                android:layout_width="match_parent"
+                android:layout_height="match_parent">
+                <ProgressBar
+                    android:visibility="gone"
+                    android:id="@+id/newDetailsLoader"
+                    style="?android:attr/progressBarStyle"
+                    android:layout_width="0dp"
+                    android:layout_height="wrap_content"
+                    android:layout_gravity="center_vertical"
+                    android:layout_weight="1" />
+            </LinearLayout>
         </LinearLayout>
         <LinearLayout
             android:layout_width="match_parent"
             android:layout_height="match_parent"
             android:orientation="vertical"
             android:padding="@dimen/static_spacing_medium">
             <TextView
+                android:id="@+id/timeAgo"
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
-                android:gravity="right"
-                android:text="3 months ago"
+                android:gravity="end"
+                android:text=""
```

Comment:
If you need to set a placeholder text to be replaced when getting the response, you can use `app:tools="@string/my_text"`.